src/lazy/text/buffer.rs

```diff
@@ -1242,16 +1242,24 @@ impl<'top> TextBufferView<'top> {
         }
     }
 
-    pub fn match_empty_arg_group(self) -> IonMatchResult<'top> {
-        recognize(pair(tag("(:"), whitespace_and_then(tag(")"))))(self)
```

Comment:
🗺️ The original impl was just returning the matched slice. Now there's a concrete type that represents an argument group, so we instantiate one of those and return it.